scripts/verify-payouts.py

```diff
@@ -0,0 +1,88 @@
+
+# ========================================= MAIN EXECUTION =========================================
+
+# Get expected total match amount
+expected_total_dai_amount = Decimal(0)
+for payout in expected_payouts:
+  expected_total_dai_amount += Decimal(payout['amount'])
+expected_total_dai_amount = expected_total_dai_amount / SCALE
+
+# Get contract instances
+w3 = Web3(Web3.WebsocketProvider(PROVIDER))
+match_payouts = w3.eth.contract(address=match_payouts_address, abi=match_payouts_abi)
+dai = w3.eth.contract(address=dai_address, abi=erc20_abi)
+
+# Get PayoutAdded events
+payout_added_filter = match_payouts.events.PayoutAdded.createFilter(fromBlock=from_block)
+payout_added_logs = payout_added_filter.get_all_entries() # print these if you need to inspect them
+
+# Get total required DAI balance based on PayoutAdded events
+total_dai_required = Decimal(0)
+for log in payout_added_logs:
+  total_dai_required += Decimal(log['args']['amount'])
```

Comment:
The payout claim for a given address can be updated (i.e. overwritten) with subsequent calls to the setPayouts function. In such a case, we'd have to track and subtract the previous value to avoid double counting those, right? Alternatively, we can filter the logs preemptively to include only the last entry made for a given account.

Reply:
Great point. I'll update the `set-payouts` to set a recipient's payout twice, then update the script to filter that out

Reply:
Added in 7778556